app/src/main/java/nu/yona/app/api/receiver/YonaReceiver.java

```diff
@@ -61,45 +61,59 @@ public void onReceive(Context context, Intent intent)
 			case AppConstant.RESTART_VPN:
 				handleRestartVPNBroadcast(context);
 				break;
+			case PowerManager.ACTION_DEVICE_IDLE_MODE_CHANGED:
+				handleDeviceDozeMode(context);
+				break;
 			default:
 				break;
 		}
 	}
 
+	@TargetApi(Build.VERSION_CODES.O)
+	private void handleDeviceDozeMode(Context context)
+	{
+		Logger.loge("BroadCast", "ACTION_DEVICE_IDLE_MODE_CHANGED");
```

Comment:
Broadcast is a single word, so please spell with lowercase c

Reply:
done. update pushed.